stl/inc/yvals_core.h

```diff
@@ -1009,6 +1011,12 @@
 #define __cpp_lib_experimental_erase_if   201411L
 #define __cpp_lib_experimental_filesystem 201406L
 
+// Functions that became constexpr in C++20, and requires is_constant_evaluated
+#ifdef __cpp_lib_is_constant_evaluated
+#define _CONSTEXPR20_ICE constexpr
+#else // ^^^ constexpr with is_constant_evaluated / vvv not constexpr without is_constant_evaluated vvv

```

Comment:
We're inconsistent about `^^^ / vvv` vs. `/` in `#else` comments, but we definitely have no occurrences with arrows on only one side of the slash.
```suggestion
#else // ^^^ constexpr with is_constant_evaluated / not constexpr without is_constant_evaluated vvv
```